infra/test_repos.py

```diff
@@ -37,17 +37,6 @@
 # the specified commits existing.
 # TODO(metzman): Fix this problem.
 TEST_REPOS = [
-    ExampleRepo(project_name='usrsctp',
```

Comment:
Why removed ? can just add a comment here to explain, no need in code.

Reply:
It's broken. I think this relied on a commit that is no longer in the tree. I think the repo's history was rewritten.